### PR TITLE
Smoke check

### PR DESCRIPTION
**Pull request by ofrancon**
Ran the notebooks from a clean env and fixed a few things

### CODE REVIEW

File: prescriptor_robojudge.ipynb [hunk 1]
```diff
@@ -72,7 +69,43 @@
     "            'C8_International travel controls',\n",
     "            'H1_Public information campaigns',\n",
     "            'H2_Testing policy',\n",
-    "            'H3_Contact tracing']"
+    "            'H3_Contact tracing',\n",
+    "            'H6_Facial Coverings']"
```
Comment (ofrancon) [inline prescriptor_robojudge.ipynb:73]: Added the latest NPI, `H6_Facial Coverings`

File: prescriptor_robojudge.ipynb [hunk 2]
```diff
@@ -72,7 +69,43 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Generate some random prescriptions\n",
```
Comment (ofrancon) [inline prescriptor_robojudge.ipynb:101]: Generating random prescriptions on the fly...

File: prescriptor_robojudge.ipynb [hunk 3]
```diff
@@ -84,13 +117,36 @@
...
     "}"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Validate the prescription files\n",
```
Comment (ofrancon) [inline prescriptor_robojudge.ipynb:133]: ... and validating them.

File: prescriptor_robojudge.ipynb [hunk 4]
```diff
@@ -84,13 +117,36 @@
    "outputs": [],
    "source": [
     "prescription_files = {\n",
-    "    'NeatExample': 'covid_xprize/examples/prescriptors/neat/test_prescriptions/pres.csv',\n",
-    "    'Random1': 'covid_xprize/examples/prescriptors/random/test/random_presc_1.csv',\n",
-    "    'Random2': 'covid_xprize/examples/prescriptors/random/test/random_presc_2.csv',\n",
-    "    'Random3': 'covid_xprize/examples/prescriptors/random/test/random_presc_3.csv',\n",
+    "#     'NeatExample': 'covid_xprize/examples/prescriptors/neat/test_prescriptions/pres.csv',\n",
+    "    'Random1': 'covid_xprize/examples/prescriptors/random/prescriptions/random_presc_1.csv',\n",
```
Comment (ofrancon) [inline prescriptor_robojudge.ipynb:121]: ... in the gitignored directory...

**Review by EKMeyerson — APPROVED** [review]
lgtm. note, one thing still to add to `prescriptor_robojudge.ipynb` is computing stringency using the given weighting before computing dominance metric.

**Pull request merged**